.vscode/launch.json

```diff
@@ -10,6 +10,15 @@
       "name": "Launch Chrome against localhost",
       "url": "http://localhost:8080",
       "webRoot": "${workspaceFolder}/client"
+    },
+    {
+      "name": "Serverless Debug",
+      "type": "node",
+      "request": "launch",
+      "cwd": "${workspaceRoot}/server",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": ["run-script", "debug"],
+      "port": 9229 
```

Comment:
Does this mean the client won't know the right port number?

Reply:
This is the port of the node debug server, not the HTTP server.